networking/understanding-networking.adoc

```diff
@@ -12,4 +12,13 @@ IP address means that Pods can be treated like physical hosts or virtual
 machines in terms of port allocation, networking, naming, service discovery,
 load balancing, application configuration, and migration.
 
+[NOTE]
+====
+Link-local IP addresses are assigned from the IPv4 `169.254.0.0/16` CIDR block. Some cloud platforms provide APIs reachable from only the link-local CIDR block.
```

Comment:
> Some cloud platforms provide APIs reachable from only the link-local CIDR block.

I understand from the word "from" in this statement that it's the cloud provider who limits the pod ability to reach the metadata service and it's not, the SDN limits it. Instead you could say that some cloud providers have APIs that listen on the link-local CIDRblock. That sounds more adequate to me

English isn't my first language, so feel free to disregard this comment if you don't agree with my interpretation.

Reply:
@juanluisvaladas you're absolutely right, and that's why trying to write this has been a challenge, beyond also not being an expert in the subject.

I'll try again. It's definitely important that the limitation is in the SDN.